demo/admin/src/products/future/ProductForm.cometGen.ts

```diff
@@ -35,6 +35,15 @@ export const ProductForm: FormConfig<GQLProduct> = {
             collapsible: true,
             initiallyExpanded: false,
             fields: [
+                {
+                    type: "asyncSelect",
+                    name: "manufacturer",
+                    rootQuery: "manufacturers",
+                    filterField: {
+                        name: "type",
```

Comment:
I currently added a not working dependency to type-field to at least show how the generated code should look like. The required manufacturerCountry field can not be generated, because of some missing async-select field features.